### DISCUSSION TITLE
Create new colors other than "Green", "Red" and "Blue" for pipeline matches

### DISCUSSION THREAD

**Discussion by airton-neto** [dagster-io/dagster#19897] [Ideas]
We'd like to have other colors marked in our assets timeline to visually mark relevant issues, not necessarily related to the code crashing.

For example:
- Our pipelines depends on the data quality in the data source. We'd like to add an "yellow" flag to these partition runs.

It would be visually shown in the timelines and easily found by querying runs, for us to re-run the partition.

![image](https://github.com/dagster-io/dagster/assets/64480652/0a2eb88c-e513-4de5-b6cf-a532939bfef7)

**Comment by braunjj** [comment]
Thanks for the suggestion @airton-neto. The lack of a yellow warning state for partitions is definitely a gap today and something I'm hoping to address soon. Are there any other colors/status indicators that you'd like to see? Is this only for partitions, or are there other parts of the product where you'd like to see more status colors?

**Reply by airton-neto** [reply]
We have this use-case case very clear, for data quality purposes (sometimes due to connection failures, for example).

I think other status could be based on Airflow run status, maybe. They have an 'skip' status too, I don't know If it makes sense to apply here
![image](https://github.com/dagster-io/dagster/assets/64480652/e6f6d3f1-dbf0-438f-b577-4bbe785547b7)

**Comment by gulivan** [comment]
This would be very useful for dbt assets: green for successful runs and tests, yellow for successful dbt runs but with failed tests, and red when both fail.
